tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py

```diff
@@ -130,4 +145,45 @@ def test_scene_move(set_up_dirs_and_db):
     assert sorted([str(a_l1_tar), str(b_l1_tar)]) == sorted(temp)
     # There is a run ard pbs file
     filename = jobdir.joinpath(PBS_ARD_FILE)
-    assert os.path.isfile(fname) is True
+    assert os.path.isfile(filename) is True
+
+
+def test_move_blocked(set_up_dirs_and_db):
+
+    # "blocked_l1_zip_path": "/home/duncan/sandbox/dea-ard-scene-select/tests/test_data/ls9_reprocessing/l1_Landsat_C2/102_076/LC91020762022178/LC09_L1TP_102076_20220627_20220802_02_T1.tar"
+    # "blocking_ard_id": "d9a499d1-1abd-4ed1-8411-d584ca45de25"
+    # "blocking_ard_zip_path": "/home/duncan/sandbox/dea-ard-scene-select/tests/test_data/ls9_reprocessing/ga_ls9c_ard_3/102/076/2022/06/27/LC09_L1TP_102076_20220627_20220627_02_T1.tar"
+    # blocking_ard_zip_path the file doesn't matter...I hope
+    blocked_scenes = [
+        {
+            "blocking_ard_id": "d9a499d1-1abd-4ed1-8411-d584ca45de25",
+            "blocked_l1_zip_path": "not used",
+            "blocking_ard_zip_path": old_fname_06_27,
+        }
+    ]
+    l1_zips, uuids2archive = move_blocked(
+        blocked_scenes, current_base_path.resolve(), new_base_path.resolve()
+    )
+
+    # Assert the dir has been moved
+    assert os.path.isfile(fname_06_27) == True
```

Comment:
do we care if the file was zero in byte size?

`
[gy5636@gadi-login-03 usgsdownloader]$ ls -la /tmp/roar
-rw-r--r-- 1 gy5636 u46 0 Jun  6 12:31 /tmp/roar
[gy5636@gadi-login-03 usgsdownloader]$ python3
Python 3.6.8 (default, Nov  8 2022, 11:32:15) 
[GCC 8.5.0 20210514 (Red Hat 8.5.0-15)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import os
>>> path="/tmp/roar"
>>> file_check = os.path.isfile(path)
>>> print(file_check)
True
>>> 
`

Reply:
> do we care if the file was zero in byte size?
not really